docs/Tutorials/Private-Network/Create-Private-Network.md

```diff
@@ -209,7 +209,7 @@ When finished using the private network, stop all nodes using ++ctrl+c++ in each
 !!!tip
 
     To restart the private network in the future, start from
-    [3. Start First Node as Bootnode](#3-start-first-node-as-bootnode).
+    [3. Start first node as a bootnode](#3-start-first-node-as-a-bootnode).
```

Comment:
This should be:

[3. Start the first node as a bootnode](#3-start-the-first-node-as-a-bootnode)